### PR TITLE
Code Quality: Clean up Files.Actions phase 1

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Motivation & Context

- Improved readability

> **Warning**<br/>Please do not review code lines that I did not changed. I cannot make more diff. If there're more codes that are preferred to change, I will do that and mark that PR as the final one for that stage.

### PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #4180
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

_None_

### CODE REVIEW

File: src/Files.App/Actions/Content/Archives/CompressIntoArchiveAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/Content/Archives/CompressIntoArchiveAction.cs:16]: I remember there was some kind of issue with the rendering using arrow instead of `{ get; }`. I'll let you know

File: src/Files.App/Actions/Content/PlayAllAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/Content/PlayAllAction.cs:20]: Shouldn't you replace `{ get; }` even here?

File: src/Files.App/Actions/FileSystem/OpenItemAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/FileSystem/OpenItemAction.cs:77]: You can remote `async` from the declaration and then return `NavigationsHelpers.Open...`

File: src/Files.App/Actions/FileSystem/OpenItemAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/FileSystem/OpenItemAction.cs:41]: As above

File: src/Files.App/Actions/FileSystem/CutItemAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/FileSystem/CutItemAction.cs:32]: You can remove `async` and then use something like:
```
    return context.ShellPage is not null
        ? UIFilesystemHelpers.CutItem(context.ShellPage)
        : Task.CompletedTask;
```

File: src/Files.App/Actions/Display/GroupAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/Display/GroupAction.cs:154]: Can you move the assignment to the constructor?

File: src/Files.App/Actions/Display/GroupAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/Display/GroupAction.cs:300]: As above

**Review by ferrariofilippo — COMMENTED** [review]
You should consider opening a different PR to refactor the initialization of `context` in most actions.
There are some little things that might be improved, but apart from that looks good

File: src/Files.App/Actions/Content/Archives/DecompressArchiveToChildFolderAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/Content/Archives/DecompressArchiveToChildFolderAction.cs:9]: This is needed

File: src/Files.App/Actions/Content/PreviewPopup/LaunchPreviewPopupAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/Content/PreviewPopup/LaunchPreviewPopupAction.cs:12]: You forgot to assign it in the constructor

**Comment by 0x5bfa** [comment]
> You should consider opening a different PR to refactor the initialization of context in most actions.

How come should I? I think it’s fine so far. Probably I will split up phase 1 PR into multiple PRs, so reviewers can review quickly and easily.

**Comment by ferrariofilippo** [comment]
> How come should I? I think it’s fine so far.

There are a lot of `Actions` which use 
`private readonly IContentApplicationContext context = Ioc.Default.GetRequired<IContentPageContext>();`

I think the assignment should be moved to the constructor

File: src/Files.App/Actions/Content/PlayAllAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Content/PlayAllAction.cs]: Fixed

File: src/Files.App/Actions/Display/GroupAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Display/GroupAction.cs]: Fixed

File: src/Files.App/Actions/Display/GroupAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Display/GroupAction.cs]: Fixed

File: src/Files.App/Actions/Content/Archives/DecompressArchiveToChildFolderAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Content/Archives/DecompressArchiveToChildFolderAction.cs:9]: Those were grayed out.

File: src/Files.App/Actions/Content/PreviewPopup/LaunchPreviewPopupAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Content/PreviewPopup/LaunchPreviewPopupAction.cs:9]: Fixed

**Comment by 0x5bfa** [comment]
> I think the assignment should be moved to the constructor

All of them were moved.

**Comment by 0x5bfa** [comment]
Ready for final review.

File: src/Files.App/Actions/FileSystem/CutItemAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/FileSystem/CutItemAction.cs:35]: CutItem is async operation

**Review by ferrariofilippo — CHANGES_REQUESTED** [review]
There might be errors with tasks not awaited

File: src/Files.App/Actions/FileSystem/OpenItemAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/FileSystem/OpenItemAction.cs:81]: Done

File: src/Files.App/Actions/FileSystem/CutItemAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/FileSystem/CutItemAction.cs:35]: Done

File: src/Files.App/Actions/Global/OpenHelpAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Global/OpenHelpAction.cs]: Done

File: src/Files.App/Actions/Navigation/CloseOtherTabsCurrentAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Navigation/CloseOtherTabsCurrentAction.cs:17]: Done

File: src/Files.App/Actions/Open/OpenSettingsAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Open/OpenSettingsAction.cs]: Done

File: src/Files.App/Actions/Navigation/NewTabAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Navigation/NewTabAction.cs:29]: Done

File: src/Files.App/Actions/FileSystem/CreateShortcutAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/FileSystem/CreateShortcutAction.cs:23]: Done

File: src/Files.App/Actions/Favorites/UnpinItemAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Favorites/UnpinItemAction.cs:66]: Done

File: src/Files.App/Actions/Favorites/PinItemAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Favorites/PinItemAction.cs:68]: Done

File: src/Files.App/Actions/Content/Archives/DecompressArchiveToChildFolderAction.cs
Comment (ferrariofilippo) [reply src/Files.App/Actions/Content/Archives/DecompressArchiveToChildFolderAction.cs:9]: My bad, the only one needed was `System.IO` (which you already added back). I selected too many rows

File: src/Files.App/Actions/FileSystem/OpenItemAction.cs
Comment (ferrariofilippo) [inline src/Files.App/Actions/FileSystem/OpenItemAction.cs]:
```suggestion
			return NavigationHelpers.OpenSelectedItems(context.ShellPage, true);
```

**Review by ferrariofilippo — COMMENTED** [review]
Some more style-related issues, but I don't think they are blockers

File: src/Files.App/Actions/FileSystem/OpenItemAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/FileSystem/OpenItemAction.cs:43]: Done

File: src/Files.App/Actions/Content/Archives/CompressIntoZipAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/Content/Archives/CompressIntoZipAction.cs:49]: Fixed

File: src/Files.App/Actions/FileSystem/AddItemAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/FileSystem/AddItemAction.cs:14]: Fixed

File: src/Files.App/Actions/FileSystem/AddItemAction.cs
Comment (0x5bfa) [reply src/Files.App/Actions/FileSystem/AddItemAction.cs:23]: Fixed

**Review by ferrariofilippo — APPROVED** [review]
LGTM

**Comment by 0x5bfa** [comment]
@yaira2 Ready to merge?

**Comment by yaira2** [comment]
@0x5bfa this looks like a big change, I'd like to get a second review before merging.

**Comment by 0x5bfa** [comment]
Should I have split up into like 3 PRs? I can do that instead of fixing conflicts and waiting reviews.

**Comment by yaira2** [comment]
We can do them together

**Comment by yaira2** [comment]
@0x5bfa besides for formatting, were there any other changes?

**Comment by 0x5bfa** [comment]
Change overview

- Moved initializers to the constructors (i.e. dependency injection initializings)
- Improved formatting
- Updated return values of Execute(); You can see this changing in ferrariofilippo's reviews

**Pull request merged**
